### PR TITLE
API provides the ability to  only view a part of premium articles

### PR DESCRIPTION
**Pull request by Cma-B**
### User Story
```
As newspaper API
In order to give a peek of the premium content
I would like to only display only the headline and an article snippet for premium articles.
```
## Changes proposed in this pull request:
* truncate premium articles for non subscriber and test 
## What I have learned working on this feature:
* how to truncate part of articles

### CODE REVIEW

**Review by Felix-Willborg — APPROVED** [review]

**Review by Alex-Bellamy — APPROVED** [review]

File: spec/requests/api/v1/articles/truncate_premium_article_for_visitor_and_non_subscriber_spec.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+RSpec.describe "GET /v1/articles for premuim", type: :request do
+  let!(:article) { create(:article, premium: true, content: 'This is the first article content' * 50) }
+
+
```
Comment (Odeane) [inline spec/requests/api/v1/articles/truncate_premium_article_for_visitor_and_non_subscriber_spec.rb:4]: Just a little space

File: db/schema.rb [hunk 2]
```diff
@@ -44,8 +44,9 @@
     t.text "content"
     t.integer "category"
     t.bigint "journalist_id"
-    t.boolean "published", default: false
```
Comment (viktorsommar) [inline db/schema.rb:47]: Why is this removed?

**Review by viktorsommar — APPROVED** [review]

**Review by Odeane — APPROVED** [review]

**Review by viktorsommar — APPROVED** [review]
